qemu/tests/qemu_guest_agent.py

```diff
@@ -261,8 +188,8 @@ def gagent_install(self, session, vm):
             if s_check == 0:
                 error_context.context("Remove the original guest agent pkg.",
                                       logging.info)
-                session.cmd("rpm -e %s" % o_check.strip())
-            self.gagent_install_cmd = "rpm -ivh %s" % self.qga_pkg_latest_url
+                session.cmd("rpm -e %s" % "qemu-guest-agent")
```

Comment:
Why do you change to use hard code `qemu-guest-agent` here ?   The old design that use the output of `gagent_pkg_check_cmd `  is more robust i think.

Reply:
with the old design, if anything changed in the future, change param `gagent_pkg_check_cmd = "rpm -q qemu-guest-agent"` is enough.

Reply:
okay, got it